db/migrate/20180823215913_move_twitch_data_out_of_users.rb

```diff
@@ -0,0 +1,35 @@
+class MoveTwitchDataOutOfUsers < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
If it's not too much work it would be nice for this to have an up and down migration so that if for some reason it needs to be reverted it can be.